README.md

```diff
@@ -26,7 +26,7 @@ golint prints out style mistakes.
 
 Golint differs from govet. Govet is concerned with correctness, whereas
 golint is concerned with coding style. Golint is in use at Google, and it
-seeks to match the accepted style of the open source Go project.
+seeks to match the accepted style of the open source Go project, [as described here](https://github.com/golang/go/wiki/CodeReviewComments).
```

Comment:
No, I don't think there *is* an "official Go style guide". The existing sentence here is intentionally loose because of that.